### PR TITLE
Task[SDK-1894]: Signed Call Android and iOS changes

### PR DESCRIPTION
**Pull request by shivamsharma2710**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2014-2021 CleverTap
```
Comment (darshanclevertap) [inline LICENSE]: Year should be 2022 @shivamsharma2710

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# clevertap_signedcall_flutter_example
+
+Demonstrates how to use the clevertap_signedcall_flutter plugin.
+
+## Getting Started
```
Comment (darshanclevertap) [inline example/README.md]: @shivamsharma2710 remove/update the Getting Started section

File: example/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Launch Screen Assets
```
Comment (darshanclevertap) [inline example/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md]: @shivamsharma2710 is this file needed?

File: LICENSE (hunk 1 above)
Comment (shivamsharma2710) [reply LICENSE]: done.

File: example/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md (hunk 3 above)
Comment (shivamsharma2710) [reply example/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md]: we've deleted it. please check

File: example/README.md (hunk 2 above)
Comment (shivamsharma2710) [reply example/README.md]: done.

**Review by darshanclevertap — APPROVED** [review]
